ecommerce/src/test/kotlin/io/hhplus/ecommerce/integration/coupon/CouponConcurrencyIntegrationTest.kt

```
@@ -71,14 +86,157 @@ class CouponConcurrencyIntegrationTest(
+                val successCount = AtomicInteger(0)
+                val failCount = AtomicInteger(0)
+
+                repeat(threadCount) { index ->
+                    executor.submit {
+                        try {
+                            couponCommandUseCase.issueCoupon(
+                                userId = 10000L + index,
+                                request = IssueCouponRequest(savedCoupon.id)
+                            )
+                            successCount.incrementAndGet()
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)